### PR TITLE
Logger fixes

### PR DESCRIPTION
**Pull request by shalvah**
Other part of a fix for #105 / logger improvements

Related:
- https://github.com/honeybadger-io/docs/pull/326
- https://github.com/honeybadger-io/honeybadger-php/pull/168

### CODE REVIEW

File: src/HoneybadgerLaravel.php [hunk 1]
```diff
@@ -43,7 +43,7 @@ public static function make(array $config): Reporter
                 'version' => self::VERSION.'/'.Honeybadger::VERSION,
             ],
             'service_exception_handler' => function (ServiceException $e) {
-                Log::error($e);
+                Log::warning($e);
```
Comment (subzero10) [inline src/HoneybadgerLaravel.php:46]: ❤️

**Review by subzero10 — APPROVED** [review]

File: src/HoneybadgerLaravel.php (hunk 1 above)
Comment (subzero10) [reply src/HoneybadgerLaravel.php:46]: Maybe we should add a comment here?
```
// Note: If you are using Honeybadger as a Logger, this exception can end up being reported to Honeybadger depending on your log level configuration.
```

**Comment by subzero10** [comment]
@shalvah I added a changelog entry to trigger the CI tests (I enabled them).

**Comment by subzero10** [comment]
@shalvah Tests are failing for L9 and L10 😢 .

**Comment by subzero10** [comment]
> @shalvah Tests are failing for L9 and L10 😢 .

I just realized that the tests are also failing on `master`. They probably broke with dependency updates and we missed them because the CI was disabled 😞

File: src/HoneybadgerLaravel.php (hunk 1 above)
Comment (subzero10) [reply src/HoneybadgerLaravel.php:48]: Comment added!

File: tests/Commands/HoneybadgerCheckinCommandTest.php [hunk 2]
```diff
@@ -8,9 +8,16 @@
 use Honeybadger\HoneybadgerLaravel\Commands\HoneybadgerCheckinCommand;
 use Honeybadger\Tests\TestCase;
 use Illuminate\Contracts\Console\Kernel;
+use Illuminate\Support\Facades\Config;
 
 class HoneybadgerCheckinCommandTest extends TestCase
 {
+    protected function setUp(): void
```
Comment (subzero10) [inline tests/Commands/HoneybadgerCheckinCommandTest.php:15]: This fixes the failing tests (they are also failing on master).

**Review by subzero10 — APPROVED** [review]

File: tests/Commands/HoneybadgerCheckinCommandTest.php (hunk 2 above)
Comment (shalvah) [reply tests/Commands/HoneybadgerCheckinCommandTest.php:15]: Ah, thanks

**Pull request merged**